src/frontend/src/lib/components/wizards/auth/views/PickAuthenticationMethod.svelte

```diff
@@ -34,8 +44,22 @@
     }
   };
 
+  const handleContinueWithOpenId = async (config: OpenIdConfig) => {
+    isAuthenticating = true;
+    const result = await continueWithOpenId(config);
+    isAuthenticating = false;
+
+    if (result === "cancelled") {
+      cancelledProviderId = config.client_id;
+      await waitFor(4000);
+      cancelledProviderId = null;
+    }
+  };
+
   const supportsPasskeys = nonNullish(window.PublicKeyCredential);
-  const showGoogleButton = canisterConfig.openid_google?.[0]?.[0];
+  const showGoogleButton =
+    canisterConfig.openid_google?.[0]?.[0] && !ENABLE_GENERIC_OPEN_ID;
+  const openIdProviders = canisterConfig.openid_configs?.[0] || [];
```

Comment:
Nit, why use `||` over `??`:
```suggestion
  const openIdProviders = canisterConfig.openid_configs?.[0] ?? [];
```

Reply:
Why would that be better? `||` has the unintended behaviour of falsy values...